test/propcheck_test.exs

```diff
@@ -42,6 +42,16 @@ defmodule PropcheckTest do
     end) == "[1,2,3]\n"
   end
 
+  test "can use assertion in forall" do
+    use PropCheck
+    assert capture_io(fn ->
+      quickcheck(
+        forall _x <- :not_ok do
+          assert false
+        end)
```

Comment:
you could add the `[:quit]` or `[:verbose]` as an option to `quickcheck`.

Reply:
That option can currently not be passed down into `forall`, as it is not part of the property itself. `quickcheck` also delegates to PropEr directly, so if we want to inject an option into the property, we probably need to change that.